app/modules/topics/actions.js

```diff
@@ -2,13 +2,17 @@
 
 import _ from 'lodash';
 
+import contentItemsModule from 'modules/content-items';
```

Comment:
I notice you doing this in a few places now... Please use the same syntax for importing modules that was used everywhere else, i.e. `import contentItems from 'modules/content-items;`. Either that or create an issue for changing it everywhere and explain why.

Reply:
I blame my IDE for this :rage: